pkg/logging/logging.go

```diff
@@ -85,6 +88,209 @@ var (
 	}
 )
 
+// Entry is a wrapper for the Logrus Entry. It exposes its public methods, but
+// also adds a functionality of storing errors and warnings as Prometheus
+// metrics. For detauls ebout Entry in Logrus see:
+// https://godoc.org/github.com/sirupsen/logrus#Entry
+type Entry struct {
+	wrappedEntry *logrus.Entry
```

Comment:
I suspect that if you make this unnamed (ie, just `*logrus.Entry` on a line), it would automatically export functions such as `Panic()`, `Fatal()` and friends, so you wouldn't need to write the boilerplate for those functions that don't need wrapping to do the metrics bits. Did you consider trying this?